styles/style.css

```
@@ -3,8 +3,90 @@ Colors to use:
 dark blue: #06365f
 light blue: #1c5380
 */
-
+* {
+  padding: 0;
+  margin: 0;
+}
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  padding: 13%;
+  background: linear-gradient( #1c5380 45%, white 0);
```

```suggestion
  background: url(../images/bgtile.png);
  background-repeat: repeat-x;
```
Podrías hacer uso de las anteriores propiedades con la finalidad de replicar el efecto de fondo observado en el screenshot del lab.